docs/decisions/001-hexagonal-architecture.md

```
@@ -0,0 +1,13 @@
+# ADR 001: Hexagonal Architecture como padrão
+
+## Contexto
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)